conans/model/graph_lock.py

```diff
@@ -83,6 +83,21 @@ def remove(self, pattern):
         self._requires = OrderedDict((k, v) for k, v in self._requires.items() if k not in remove)
         return remove
 
+    def update(self, refs, name):
+        if not refs:
+            return
+        for r in refs:
+            r = RecipeReference.loads(r)
+            new_reqs = {}
+            for k, v in self._requires.items():
+                if r.name == k.name:
+                    ConanOutput().info(f"Replacing {name}: {k.repr_notime()} -> {repr(r)}")
```

Comment:
Yes, maybe, but this could be a wider feature, to have formatter output, but also for ``remove`` too, for example? Probably can wait until it is requested.